Examples/Scripts/Python/full_chain_itk.py

```diff
@@ -55,7 +55,7 @@
     s,
     trackingGeometry,
     field,
-    # ParticleSelectorConfig(eta=(-4.0, 4.0), pt=(1.0 * u.GeV, 10.0 * u.GeV), removeNeutral=True),
+    # ParticleSelectorConfig(eta=(-4.0, 4.0), pt=(150 * u.MeV, None), removeNeutral=True),
```

Comment:
This was and stays commented out?

Reply:
Yes, it's just an example in case people want to run a ttbar example. We decided we didn't want command-line options, and I think a separate file is overkill, so I think this is a reasonable way to document this.
I tested it by uncommenting.